lib/rs/Cargo.toml

```diff
@@ -2,8 +2,11 @@
 
 name = "thrift"
 version = "0.0.1"
-authors = ["Simon Génier <s@simon.coffee>", "Maxim Golov <maxim.golov@gmail.com>"]
+authors = ["Jonathan Reem <jonathan.reem@gmail.com>",
+           "Simon Génier <s@simon.coffee>",
+           "Maxim Golov <maxim.golov@gmail.com>"]
```

Comment:
Not sure what is the usual Rust practice, but I'd suggest we stick to either alphabetical or chronological order here (which will have the same result in this case).